migration/sql-files/012-add-user-feature-group.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE users ADD COLUMN feature_level TEXT;
```

Comment:
Would we want to make it `not null` and also add a default value to this column for all existing rows?

Reply:
no, I don't think so: it's like a program that users can opt-in/opt-out, but it's not mandatory, so I'd rather keep the column `nullable` for now, if it's ok for you.

Reply:
Isn't there a default group where all of us would be in , after we introduce this?

Reply:
weel, in that case, that would be a group in which we only use "released" features. So that could be the default level, if you prefer it that way.